### PR TITLE
Enable self-collisions in Gazebo

### LINKED ISSUE #121: Update marimba URDF for self coliisions
Update the `marimba.urdf.xacro` to avoid any objects intersecting, and set gazebo selfCollision to `true` for everything in `marimba.urdf.xacro`. This way, the marimba and ur5 can be loaded as one URDF and gazebo will be aware of collisions between the arm/mallet and the marimba.

### PR DESCRIPTION
**Pull request by iibrahimli**
Enable self-collisions in Marimba UR5 URDF

Also (unrelated change, I know) reverts `load_robot_description.launch` to `marimbabot_ur5_upload.launch` to keep it consistent with TAMS naming.

Closes #121

### CODE REVIEW

File: marimbabot_description/urdf/marimba.urdf.xacro [hunk 1]
```diff
@@ -130,12 +130,12 @@ Changelog:
...
     </link>
 
-    <xacro:if value="$(arg bars_fixed_joint)">
+    <xacro:if value="${arg bars_fixed_joint}">
         <xacro:tams_fixedjoint name="bar_${note}/joint" parent="${parent}" child="bar_${note}" >
             <origin xyz="${pos}" rpy="0 0 0" />
         </xacro:tams_fixedjoint>
     </xacro:if>
-    <xacro:unless value="$(arg bars_fixed_joint)">
+    <xacro:unless value="${arg bars_fixed_joint}">
```
Comment (Juphex) [inline marimbabot_description/urdf/marimba.urdf.xacro:138]: Please revert. Doesn't work with curly brackets. At least not for me 😅

File: marimbabot_description/urdf/marimba.urdf.xacro [hunk 2]
```diff
@@ -130,12 +130,12 @@ Changelog:
     </collision>
     </link>
 
-    <xacro:if value="$(arg bars_fixed_joint)">
+    <xacro:if value="${arg bars_fixed_joint}">
```
Comment (Juphex) [inline marimbabot_description/urdf/marimba.urdf.xacro:133]: same

File: marimbabot_description/urdf/marimba.urdf.xacro (hunk 1 above)
Comment (Juphex) [reply marimbabot_description/urdf/marimba.urdf.xacro:138]: At least when I start it in rviz.

**Review by Juphex — CHANGES_REQUESTED** [review]
As I understood the complete model is now enabled to have a collision. Doesn't it make more sence to enable it only for specific parts like the bars? at least for the start

**Comment by iibrahimli** [comment]
> As I understood right now: the complete model is now enabled to have a collision. Doesn't it make more sence to enable it only for specific parts like the bars? at least for the start

@Juphex In that case mallet/arm would go right through side boxes or body, which is not the desired behavior.

**Review by Juphex — APPROVED** [review]

**Comment by Juphex** [comment]
works

**Pull request merged**